MeetU/MeetU/Scripts/Controllers/Meetup/edit.js

```diff
@@ -0,0 +1,48 @@
+﻿(function () {
+    "use strict";
+    angular
+        .module('meetupModule')
+        .controller('MeetupEditController', MeetupEditController)
+    MeetupEditController.$inject = ["$rootScope", "$log", "$resource", "$location", "$scope", "$timeout"];
+    function MeetupEditController($rootScope, $log, $resource, $location, $scope, $timeout) {
+        var vm = this;
+        $rootScope.$on("UpdateMeetup", function (event, message) {
```

Comment:
it is not right. you should use service to pass value, and use events to update values in index